doc/source/parameters/cfd/multiphysics.rst

```diff
@@ -40,6 +41,8 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
 
    * ``buoyancy force``: controls if the buoyancy force is taken into account in the Navier-Stokes equations. The buoyancy force is calculated using the Boussinesq approximation.
 
+   * ``DCDD shock capture``: The discontinuity-capturing directional dissipation method is used the stabilize the heat transfer equation. For more information, see Tezduyar, T. E. (2003). 
```

Comment:
```suggestion
   * ``DCDD shock capture``: Applies the discontinuity-capturing directional dissipation (DCDD) stabilization term on the heat transfer equation. For more information, see Tezduyar, T. E. (2003).
```

Reply:
If you change the section, please don't forget to change the documentation accordingly.